flask_log.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""
+Setup a nice formatter, configurable level, and syslog handler for logging.
+"""
+
+import logging
+from logging.handlers import SysLogHandler
+import os
+import platform
+
+
+class Logging(object):
+    """Configure flask logging with nice formatting and syslog support."""
+
+    def __init__(self, app=None):
+        """Boiler plate extension init with log_level being declared"""
+        self.log_level = None
+        self.app = app
+        if app is not None:
+            self.init_app(app)
+
+    def init_app(self, app):
+        """Setup the logging handlers, level and formatters.
+
+        Level (DEBUG, INFO, CRITICAL, etc) is determined by the
+        app.config['FLASK_LOG_LEVEL'] setting, and defaults to
+        ``None``/``logging.NOTSET``.
+
+        """
+        config_log_level = app.config.get('FLASK_LOG_LEVEL', None)
+
+        # Set up format for default logging
+        hostname = platform.node().split('.')[0]
+        formatter = (
+            '[%(asctime)s] %(levelname)s %(process)d [%(name)s] '
+            '%(filename)s:%(lineno)d - '
+            '[{hostname}] - %(message)s'
+        ).format(hostname=hostname)
+
+        config_log_int = None
+        set_level = None
+
+        if config_log_level and not set_level:
```

Comment:
thanks, that is a hold over from a version that accepted a set_level as a parameter